app/src/split/bluetooth/central.c

```diff
@@ -685,6 +693,32 @@ int zmk_split_bt_invoke_behavior(uint8_t source, struct zmk_behavior_binding *bi
     return split_bt_invoke_behavior_payload(wrapper);
 }
 
+static zmk_hid_indicators hid_indicators = 0;
+
+static void split_central_update_indicators_callback(struct k_work *work) {
+    zmk_hid_indicators indicators = hid_indicators;
+    for (int i = 0; i < ZMK_SPLIT_BLE_PERIPHERAL_COUNT; i++) {
+        if (peripherals[i].state != PERIPHERAL_SLOT_STATE_CONNECTED) {
+            continue;
+        }
+
+        int err = bt_gatt_write_without_response(peripherals[i].conn,
+                                                 peripherals[i].update_hid_indicators, &indicators,
+                                                 sizeof(indicators), true);
```

Comment:
See https://github.com/chrisandreae/zmk/commit/bc824de85811a62018c33fe3828a1f0b51453a3f from the MoErgo folks for a small safety check that would be good here, if this gets called mid-attribute discovery.